pom.xml

```diff
@@ -13,7 +13,7 @@
   <inceptionYear>2004</inceptionYear>
 
   <prerequisites>
-    <maven>2.2.1</maven>
+    <maven>3.2.2</maven>
```

Comment:
I assume you chose 3.2 instead of 3.3 because you wanted to retain compatibility with Java 6 for those plugins which may still be running with Java 6.

Was there a reason you didn't choose [3.2.5](https://maven.apache.org/docs/3.2.5/release-notes.html) instead of 3.2.2?  The [3.2.5 release](https://maven.apache.org/docs/history.html) was over two years ago, and it supports Java 6.